runtime-modules/content/src/nft/types.rs

```diff
@@ -233,18 +242,15 @@ impl<
     }
 
     pub(crate) fn with_bid(self, amount: Balance, bidder_id: MemberId, block: BlockNumber) -> Self {
-        let (auction_duration, end) = if self.end.saturating_sub(self.extension_period) < block {
-            (
-                self.auction_duration.saturating_add(self.extension_period),
-                self.end.saturating_add(self.extension_period),
-            )
+        // sniping extension logic
```

Comment:
I would explicitly write out the logic here in the comment. If bid block is in `[end - extension_period, end)` then new `end` is `end + extension_period`